switcheroo/server/retrieve_public_keys.py

```diff
@@ -7,8 +7,8 @@
 
 def get_public_keys_local(connecting_user: str, ssh_home_dir: str) -> str:
     host_name = socket.getfqdn()
-    key_dir = f"{ssh_home_dir}/{host_name}"
-    key_path = f"{key_dir}/{connecting_user}-cert.pub"
+    key_dir = f"{ssh_home_dir}/{host_name}/{connecting_user}"
+    key_path = f"{key_dir}/key-cert.pub"
     if not os.path.isdir(key_dir):
```

Comment:
I feel like I reviewed this already and we didn't act on the review - so here it goes again - don't test and act - rather act and handle the exception

also use `pathlib`